### ISSUE TITLE
Default values don't appear in the output

### ISSUE THREAD

**Issue by hegemonic** [hegemonic/jsdoc-baseline#139]
For the following code, I would expect Baseline to show the default value of `foo` in the output, but it doesn't:

```js
'use strict';

/** @module baz */

/**
 * @param {boolean} [foo=true] It's foo!
 */
exports.bar = function(foo) {};
```

**Comment by hegemonic** [comment]
Fixed on master.

**Comment by hegemonic** [comment]
Reopening. We show the "Default value" list item for the following example, but we don't actually show the default value, even though it's captured in the doclet:

```js
/**
 * Module representing the word of the day.
 * @module wotd
 */

/**
 * The word of the day.
 * @default
 */
module.exports = 'perniciousness';
```

**Comment by Alaneor** [comment]
Please also consider perhaps a tricky situation where I want to document a default value of being `null`:

`@default   null`

**Comment by hegemonic** [comment]
Fixed by f20b0c2fb5444c73365e77d5d4d8ea6eb51764fc.
